pages/common/brew.md

```diff
@@ -0,0 +1,32 @@
+# brew
+
+> Package manager for macOS and Linux.
+> More information: <https://brew.sh>.
+
+- Search for available formulae (i.e. packages) and casks (i.e. native packages):
+
+`brew search {{text}}`
+
+- Install the latest stable version of a formula or cask (use `--devel` for development versions):
+
+`brew install {{formula}}`
```

Comment:
It might be a good idea to switch the positions of these two, because `install` seems more essential than `search` 🤔

Reply:
Done in fb8a8d39.